webpack.config.base.js

```diff
@@ -43,6 +46,9 @@ module.exports = {
     new ExtractTextPlugin({
       filename: '[name].css',
       allChunks: true
+    }),
+    new webpack.EnvironmentPlugin({
+      BACKEND_URL: process.env.BACKEND_URL,
```

Comment:
Obviously the JS running in people's browsers won't have access to the server's env vars, so I don't quite get this.